lblod/app-digitaal-loket
# Task: Add opvolgmoment 2024 for LEKP 1.0

## ID
 DGS-85

 ## Description
PR adds a new `opvolgmoment 2024` for LEKP 1.0, to not lose previous data in the opvolgmoment. With startTime set to 08 December 2023. And endTime set to 01 March 2024.

 ## Type of change


 ## How to test

Visit subsidiebeheer in loket and verify the subsidies of LEKP 1.0 now have a new step called `opvolgmoment 2024` with a startTime of 08 december 2023, and an endTime of 01 March 2024.
